### PR TITLE
Fix test according to stderr format of go (unknown) command

### PR DESCRIPTION
**Pull request by k1LoW**

### CODE REVIEW

**Comment by ivy** [comment]
@k1LoW It looks like this only fixes the build for `tip` (as specified in the project's [`.travis.yml`](https://github.com/mattn/go-shellwords/blob/master/.travis.yml#L3)). I think it would be better to write the test in a way that isn't so tightly coupled to the locally installed Go release, to allow `go test`ing elsewhere.

**Comment by k1LoW** [comment]
@ivy 

>  It looks like this only fixes the build for tip

You are right 👍 Thank you for the feedback.

I just fixed it because I felt that the failure condition was not good.

- Go runtime output depends on Go version
- Other command output depends on command version, OS, Localization

mmm...  🤔

**Comment by ivy** [comment]
@k1LoW I think the easiest fix would be to convert that expected string into a regexp. Like you mentioned though, I'm not sure how much localization changes that output.

I think we should also add Go `1.11.x` to the [Travis CI config](https://github.com/mattn/go-shellwords/blob/525bedee691b5a8df547cb5cf9f86b7fb1883e24/.travis.yml#L2-L3) or whatever the version prior to the build breakage was.

Alternatively, if the output is drastically different, it might be better to use a test helper, [similar to os/exec's test suite](https://npf.io/2015/06/testing-exec-command/).

File: shellwords_test.go [hunk 1]
```diff
@@ -110,13 +110,18 @@ func TestBacktick(t *testing.T) {
 }
 
 func TestBacktickError(t *testing.T) {
+	_, err := shellRun("go build -o ./_testdata/stderr ./_testdata/stderr.go")
```
Comment (ivy) [inline shellwords_test.go]: Love your use of `_testdata/`! It's very clean. :+1:

**Comment by ivy** [comment]
@mattn I believe this PR is suitable for merging. Hope you can have a look soon. 😄

**Pull request closed without merging**